conda/_vendor/tqdm/utils.py

```diff
@@ -368,6 +368,8 @@ def _screen_shape_linux(fp):  # pragma: no cover
                 return [int(os.environ[i]) - 1 for i in ("COLUMNS", "LINES")]
             except KeyError:
                 return None, None
+            except ValueError:
+                return [0, 0]
```

Comment:
Is there a reason for returning `[0, 0]` rather than `(None, None)`? Looking at the [`tqdm.__init__(...)` docstring](https://github.com/conda/conda/blob/f47dc5e6f/conda/_vendor/tqdm/std.py#L858-L863) where `_screen_shape_wrapper(...)` gets called, this would (likely) suppress any progress meter, rather supporting dynamic resizing at a later time.

Reply:
In the earlier code before the refactoring, `1` was used as the default value. I tried to do the same here.